src/realmGeolContinental.ttl

```diff
@@ -145,6 +156,21 @@ soreagcont:StableContinent rdf:type owl:Class ;
                                      ] ;
                      rdfs:label "stable continent"@en .
 
+###  http://sweetontology.net/realmGeolContinental/Supercraton
+soreagcont:Supercraton rdf:type owl:Class ;
+                     rdfs:subClassOf soreag:GeologicProvince ,
+                                     [ rdf:type owl:Restriction ;
+                                       owl:onProperty sorel:hasPart ;
+                                       owl:allValuesFrom soreagcont:Craton
+                                     ] ,
+                                     [ rdf:type owl:Restriction ;
+                                       owl:onProperty sorel:hasPart ;
+                                       owl:minCardinality "2"^^xsd:int
+                                     ] ;
+                     dcterms:source <http://doi.org/10.22459/SN.08.2012> ;
+                     rdfs:subClassOf soreag:GeologicProvince ;
```

Comment:
Is every Supercraton also a GeologicProvince? Is this true @smrgeoinfo ?

Reply:
A supercraton consists of two or more (largely Archaean) cratons (Blewett, 2012: https://press.anu.edu.au/publications/shaping-nation), and as cratons are tectonic provinces, a supercraton consists of as many provinces as there are cratons in the supercraton in question. And, if the truth be known, it we dig a little deeper in a craton, we would undoubtedly find provinces within provinces (if they are still recognisable as such, considering their age and complex, multiphase tectonic history). I will leave Nick to deal with how best to approach the multi-province aspect.

Reply:
question remains: is a supercraton a craton in which case it should be a subClass, and if a craton is a GeologicProvince, then a supercraton is a GeologicProvince. 

As I read the proposed definitions a craton is a "part of Earth's continental crust that has attained stability and has been little deformed for a prolonged period"  so it sounds like a supercraton might or might not be a craton.   Western North America basement (east of Cordillera) is an amalgamation of Archean through Early Proterozoic terranes, amalgamated by Early Proterozoic time, and certainly behaving as a craton through the Early Paleozoic, so perhaps a 'supercraton'? This 'craton' has see considerable tectonic activity on the western margin since then, so is it still a 'craton' or has the craton changed extent? is it still a super craton?

Reply:
A supercraton would be a geological province, but, Stephen, what are you suggesting to be the subclass—the craton or the supercraton? We should not be here to question what others, in published literature, have delineated a supercration to be, and if their usage of the term is valid.
I am no expert on cratons. I am just referring to the Glossary and other sources. But, in terms of the very old ages of the features that we are discussing here, I imagine that a craton can form and be stable and little deformed for a long period of (geological) time, thus satisfying the criteria for being termed a craton. But, then, if after such a long period, this stability is interrupted by tectonism or whatever, causing, say, an amalgamation of cratons, should the amalgamating cratons then cease to be termed cratons? Should the term supercraton be discarded as the once-stable cratons that comprised it have ceased to be cratons (because of tectonic amalgamation), and a new craton (not a supercraton) is formed (which then needs to be stable and little deformed for a succeeding interval)? Again, should we not comply with the published literature.

I also assume that a craton, although stable, would not be isolated from any major tectonic activity, as tectonism even at plate margins can affect a whole plate. Does not the qualification ‘little deformed’ cater for this? I welcome any definition to satisfy all queries. In achieving this, it seems that we may deviate from the published definitions and develop our own. If this is the case, we can overcome issues of copyright and links to copyrighted material (see my comments re this elsewhere here).

Reply:
I agree that to the extent possible its good to define terms to be logically consistent with common usage.  Problems arise when common usage is inconsistent. I think the role of information resources like SWEET is to provide some way to disambiguate usages (much like wikipedia...) with clear definitions of the various possible interpretations of a term, with identifiers to distinguish the meanings. 

As far as Craton/supercraton, it think perhaps best if they are both kinds of 'tectonic feature', (or 'geologic feature' or 'tectonic entity', all possible labels for the concept), and  _Supercraton 'hasPart' Craton_. This seems compatible with a quick google  survey of usage in the literature, e.g.  https://ui.adsabs.harvard.edu/abs/2004AGUSM.T41C..01B/abstract, http://people.earth.yale.edu/sites/default/files/files/Evans/54_13a-Smirnov%2BZimgarn.pdf and the definition (originally?) propsed by [Bleeker 2003](https://www.sciencedirect.com/science/article/pii/S0024493703001099):  "_large ancestral landmasses of Archean age with a stabilized core that on break-up spawned several independently drifting cratons_"

Reply:
So short answer to @dr-shorthair original question, yes I think every supercraton could be considered a 'geologic province'.

Reply:
Both kinds of 'tectonic entity' is OK with me, rather than classifying under 'province', but along with provinces which are also tectonic entities, as suggested in my previous post??